app/models/product.rb

```diff
@@ -0,0 +1,12 @@
+class Product < ApplicationRecord
+  has_one_attached :photo
+
+  def self.ransackable_attributes(auth_object = nil)
```

Comment:
def self.ransackable_attributes(auth_object = nil)
    ["name", "price","total_quantity"]
  end
  
  def self.ransackable_associations(auth_object = nil)
    ["photo_attachment", "photo_blob"]
  end
  
  Add a single line space in between two methods